docs/docs/api/api/puffer-client-helpers.md

```diff
@@ -35,7 +35,7 @@ configuration
 
 `object`
 
-The public client created with viem.
+The public client was created with viem.
```

Comment:
This was correct. Please revert this change. And API docs are generated from comments in the code, so updating it here won't work.

Reply:
done